torchtune/models/convert_weights.py

```diff
@@ -246,6 +246,29 @@ def tune_to_peft_adapter_config(
     return adapter_config
 
 
+# When we only save adapter weights, we need to convert back from peft format to tune format
```

Comment:
I use this when trying to load adapter config weights back. Will add sample code for what I got working later

Reply:
As in, resuming training in torchtune using adapter weights saved from a previous torchtune LoRA run? How come that wasn't working?